### PR TITLE
Ephemeral location: do not relay on nil value when it's not an image

### PR DESCRIPTION
**Pull request by mikeger**
### Issues

The logic was relying on the old protobuf compiled classes behavior, where when the field was not initialized it was returning nil. In the new Swift protobuf it's returning a shell object.

### CODE REVIEW

**Comment by vytis** [comment]
Would be good to add a test for this, but not sure if worth creating a new class if nothing like this exists...

File: Source/Utilis/Protos/GenericMessage+Helper.swift [hunk 1]
```diff
@@ -61,7 +61,12 @@ extension GenericMessage {
         case .image(let data):
             return data
         case .ephemeral(let data):
-            return data.image
+            switch data.content {
```
[the remark this answers is not in the document]
Comment (mikeger) [reply Source/Utilis/Protos/GenericMessage+Helper.swift]: It's not super straightforward for me, ephemeral has the object as associated data, not another enum, so it's not an enum with enum:

```swift
public struct GenericMessage {
  public enum OneOf_Content : Equatable {
        ...
        case ephemeral(WireProtos.Ephemeral)
  }
}
```

```swift
public struct Ephemeral {
  public var content: WireProtos.Ephemeral.OneOf_Content?

  public enum OneOf_Content : Equatable {
        ...
        case location(WireProtos.Location)
  }
}

```

[the remark this answers is not in the document]
Comment (vytis) [reply Source/Utilis/Protos/GenericMessage+Helper.swift]: Ah yeah, makes sense. Just wanted to make sure we don't make the same error when adding more properties to generic message as this seems very copy-pasteable 🙈

**Comment by mikeger** [comment]
@vytis added the test

**Comment by mikeger** [comment]
@zenkins release patch

**Pull request merged**